src/Xml/XmlDocument.cpp

```diff
@@ -29,7 +29,9 @@ XmlDocument::XmlDocument() :	XmlNode(XmlNode::XML_DOCUMENT),
  *
  * \param documentName	Name of the document.
  */
-XmlDocument::XmlDocument(const std::string& documentName) :	XmlNode(XmlNode::XML_DOCUMENT),
+XmlDocument::XmlDocument(const std::string& documentName)
+: XmlNode(XmlNode::NodeType::XML_DOCUMENT)
+,
```

Comment:
The formatting here looks kind of funny, though perhaps we should address that as a separate issue. I've opened issue #242 to track this.

Reply:
I blame Clang-Tidy and the adherence to tabs. :P Really it was me rapidly cycling through the output errors list and pasting the enum scope: F8, Ctrl+V. F8, Ctrl+V, etc. Will fix in #242.